### ISSUE TITLE
Collection of malkəʾ ID

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#1381]
Following @eu-genia 's suggestion in https://github.com/BetaMasaheft/Manuscripts/pull/286, I consider creating a Generic ID for non-standard collection of malkəʾ-hymns. The suggestion is, I believe, to have LIT1850Malkea for collections close to the current printed edition, and a further ID for those widely differing from it, such as the texts attested in BLorient579. I think this is logical, but don't know the manuscript tradition very well. Can we speak of a divide between the texts as presented in the edition and all others, or is the process a more continuous one? 

Another related question: There is the ID LIT3185Mazgab for "Mazgaba malkǝʾ", attested only in an inventory in BLadd16197. Any ideas whether this refers to a Gubāʾe malkəʾ like collection (standardised or not...) or whether this might be a distinct text?

**Comment by DenisNosnitsin1970** [comment]
To respond to your first question, we need to study the manuscript tradition of Mälke'a guba'e. the printed version is perhaps the most authoritative today, and possibly the most complete, but in the manuscript tradition it will probably not be the standard. Then, if you create ID for non-standard collections of MG, all collections in manuscript will possibly be non-standard (the case of Psalter is somewhat different, as we do have a strong tradition of the "standard Psalter", and the number of mss with "non-standard" versions is small). If this is acceptable and you will not disturbed by this fact, you can go ahead. Otherwise someone can make a short survey and try to find at least two or three witnesses with the same collection, and accept this as the "standard one" (maybe, you will find indeed witnesses with the same collection as in the printed book, even better).

**Comment by thea-m** [comment]
Thank you! I am disturbed by it, because even though we need some fixed references to work around, it would be good to work starting from the manuscript tradition. 
I can look into it in more detail, but, as you might guess, not right now :) If no one else is interested to do this now, I'll just assign myself and keep the issue open.

**Comment by DenisNosnitsin1970** [comment]
It would be good, yes. As to the LIT3185Mazgab for "Mazgaba malkǝʾ", I would not be deterred by the title. It may be a type of the same collection, you need to check the content a little bit. I wonder if the title "Malke'a guba'e" is indeed well attester in the mss, or it got spread as the title of the printed collection (and became a generic title).

**Comment by thea-m** [comment]
Yes, this was my suspicion. as I wrote this title is only attested in an inventory, so it is not possible to check the content...

**Comment by DenisNosnitsin1970** [comment]
I would assume, it referes to a Malke'a guba'e.

**Comment by eu-genia** [comment]
Unless we create additional list of authority files for traditional "labels" I believe we do need at least one "abstract" ID to start working with, where we would collect all labels possible malk collections could have circulated under, in whichever selection or sequence. Once there is some more data we could do the work on disambiguation. But I understand from long discussions with @abausi that the research between the traditional labels and real works should be one of our aims.

**Comment by abausi** [comment]
I think this can be done: also using labels such as "malkXXX" os something of this kind. The research on labels, works, collections etc. remains one of the main aims of BM, no doubt, but one can start to collect the evidence and this intermediate stage is absolutely necessary. I agree with Zhenia.

**Comment by thea-m** [comment]
Wonderful, as I wrote previously, I right now have no time to carry this out thoroughly (and don't believe in rushing such things). I will keep this issue open and won't be sad if someone assigns themselves.

**Comment by thea-m** [comment]
@smaugustine maybe I can move the responsability for this issue to you?

**Comment by smaugustine** [comment]
@thea-m Sure — it was reading through this thread helped lead me to my idea for a topic.

**Comment by eu-genia** [comment]
anything to be done here? has this been solved?

**Comment by smaugustine** [comment]
My dissertation proposes groups of manuscripts with overlapping anthologies of malke'. This issue was actually part of the inspiration for it! Once I submit I can incorporate it into the discussion here. To be clear, there's not a meaningful distinction between the titles መልክአ፡ ጉባኤ፡, ጉባኤ፡ መልክእ፡, መዝገበ፡ መልክእ፡, or መጽሐፈ፡ መልክእ፡, but it is possible to recognize families that prefer certain malke' over others.

**Comment by eu-genia** [comment]
@smaugustine is there anything we need or can do to close the issue? are we fine as we are? thank you!

**Comment by smaugustine** [comment]
I would agree with deleting `LIT3185Mazgab`. Since the Gəʿəz labels do not themselves indicate any difference in the contents it would be sensible to include all the potential titles in one record.

It might be useful to incorporate a distinction made in my dissertation: anthologies vs collections. As I define them:

- An anthology: contains (1) four or more malkəʾ/quasi-malkəʾ (2) representing two or more types of subjects (ie Trinitarian, angelic, Marian, etc)
- A collection: (1) contains predominantly malkəʾ or quasi-malkəʾ and (2) includes 3 or more malkəʾ selected based on a common theme

Thus MSS like BL Or. 573, 574, 575 are anthologies whereas MSS UNESCO 8-13, Bodl. Aeth. e. 21, and BL Add. 16216 are collections.

**Comment by eu-genia** [comment]
We can use LIT1850Malkea#type1ant and LIT1850Malkea#type2coll where we want to separate the two types, thank you!
